src/test/java/com/puppycrawl/tools/checkstyle/api/ScopeTest.java

```diff
@@ -74,15 +74,15 @@ public void testMixedCaseSpaces() {
 
     @Test
     public void testMixedCaseSpacesWithDifferentLocales() {
-        final Locale[] differentLocales = {new Locale("TR", "tr") };
-        final Locale defaultLocale = Locale.getDefault();
```

Comment:
@romani We went through this before.
This is breaking the test as we have to save `defaultLocale` before we modify it. By the time `finally` is called, it is already modified.